reference/recency-surveillance.v1.json

```diff
@@ -223,139 +228,164 @@
                           "Others (Specify)"
                         ]
                       },
-                      "reason_for_testing_other": {
+                      "special_category_other": {
                         "type": "string",
                         "default": "text"
                       },
-                      "reason_for_testing": {
+                       "first_time_hiv_test": {
+                        "description": "Has never tested before",
                         "type": "string",
                         "enum": [
-                          "Assisted Patner Notification (APN)",
-                          "Index Client Testing other than APN",
-                          "PrEP",
-                          "PEP",
-                          "HIV Self-Test Positive",
-                          "Inconclusive HIV Result",
-                          "Self Initiative",
-                          "PMTCT",
-                          "Others"
+                          "No",
+                          "Yes"
                         ]
                       },
-                      "community_testing_entry_point_other": {
+                      "last_hiv_test_date": {
+                        "description": "Must be YYYY-MM-DD format",
```

Comment:
@ODORA0 update the description to match the default format and also an example date to illustrate

Reply:
@ssmusoke Done